users/views.py

```diff
@@ -154,6 +155,15 @@ def user_create_confirm(request, token):
         user.confirm_token = ''
         user.is_active = True
         user.save()
+        user.circle_set.create(
+            name='Me',
+            is_editable=False
+        )
+        user.circle_set.create(
+            name='Public',
+            is_editable=False
+        )
```

Comment:
I think this is a massive overhead. It would better be handled in the code instead of the database. When displaying the circle selector or when checking if a post is visible, it is enough to check for these two edge cases.